src/BlazorWebView/src/SharedSource/BlazorWebViewDeveloperTools.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.AspNetCore.Components.WebView.Maui
 #endif
 {
 	internal class BlazorWebViewDeveloperTools
-	{
-		public bool Enabled { get; set; } = false;
-	}
+{
+	public bool Enabled { get; set; } = false;
+}
```

Comment:
This formatting is not correct. Why is the tool not using the csproj defines?

Reply:
This does seem like a bug (or limitation) of the formatter... can we tell it to exclude certain files/folders that we know don't work?